### ISSUE TITLE
Debugging in iOS does not work if application id has a dash in it

### ISSUE THREAD

**Issue by brianchirls** [NativeScript/ios-runtime#77]
My application id looks kinda like this: `com.example.hello-world`. When I run the debug command, I get this error:

```
Invalid application id: com.example.helloworld. All available application ids are: 
...
com.example.hello-world
....
```

**Comment by brianchirls** [comment]
The discrepancy seems to happen when the project is created. To reproduce:

1. Run `tns create hello-world`.

The project is created in a folder called `hello-world`, but the project id inside .tnsproject is `org.nativescript.helloworld`.

2. `cd hello-world`
3. `tns platform add ios`
4. `tns debug ios --debug-brk`

Same error message as above.

Changing the id inside .tnsproject to include the dash fixes the problem.

**Comment by PanayotCankov** [comment]
We had an issue that the Xcode project did not pick propertly the id from .tnsproject. The issue may be resolved already. If not we should provide a fix for v1.0

**Comment by jasssonpet** [comment]
Fixed in https://github.com/NativeScript/nativescript-cli/pull/450
